### DISCUSSION TITLE
In the need of more url fields on the right pane!

### DISCUSSION THREAD

**Discussion by dtmf7133** [community/community#60353] [Repositories]
### Select Topic Area

Product Feedback

### Body

As from subject Id like to have the possibility to associate more "visible" websites to a repo.
I understand that cheking urls at your side can be trivial..
But I find a limitation having only one link, you have the home of the project to let others download the code, you have the instance of the web app running, maybe a second good example..
There are reasons to think positive regarding this stuff?

Daniele Bonini

**Comment by ronakjkothari** [comment]
While I do agree it would be nice to showcase more than one website, I believe that this method of showcasing only one encourages users to build a portfolio that showcases all their websites. I find that to be nice, as a portfolio goes a long way in a programmer's career.
